requirements.txt

```diff
@@ -1,5 +0,0 @@
-requests
-python-dotenv
-pytest
```

Comment:
Do we need to account on `pytest` on poetry configuration file?

Reply:
`pytest` is present in the `dev-dependencies` section of the `pyproject.toml`